tests/v2/validation/rbac/rbac_etcd_test.go

```diff
@@ -0,0 +1,132 @@
+package rbac
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/rancher/rancher/tests/v2/validation/provisioning/rke2"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type ETCDRbacTestSuite struct {
```

Comment:
can we update the test names and filename to include `snapshot` or `backup` to be consistent with other test files i.e. `rbac_etcd_backup_test.go` ... `ETCDRbacBackupTestSuite`

Reply:
done